spark_pipeline_framework/transformers/fhir_receiver/v1/fhir_receiver.py

```diff
@@ -493,9 +493,17 @@ def _transform(self, df: DataFrame) -> DataFrame:
                 else str(parameters["flow_name"])
             )
             if additional_request_headers:
-                additional_request_headers.update({"User-Agent": user_agent_value})
+                additional_request_headers.update(
+                    {
+                        "User-Agent": user_agent_value,
+                        "Origin-Service": str(parameters["flow_name"]),
```

Comment:
Please check the formatting in the ticket description. It should be like {repository:pipeline_name}